source/sgp_mode/SGPWorld.h

```diff
@@ -164,7 +166,8 @@ class SGPWorld : public SymWorld {
 
   
   emp::WorldPosition SymDoBirth(emp::Ptr<Organism> sym_baby, emp::WorldPosition parent_pos) override;
-  int GetNeighborHost (size_t id, emp::Ptr<emp::BitSet<64>>);
```

Comment:
Add a constexpr variable somewhere for this bitset size? Probably want to define it in whatever location is dictating the choice of value.